### ISSUE TITLE
Implement broadcast

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Chobby#443]
See https://github.com/ZeroK-RTS/Zero-K-Infrastructure/issues/1883

**Comment by gajop** [comment]
Doesn't it work? I saw a popup yesterday when it got restarted due to the Windows update.

**Comment by Licho1** [comment]
Yes but popup is gone and disappears when server disconnect. It should stay there and wait for people to acknowledge it (click 'ok" or something)

**Comment by KingRaptor** [comment]
Bumping this because our server update processes are apparently really terrible at communicating what is going on to people.
